src/styles/blocks/star.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+  width: 96px;
+  margin-left: 18px;
```

Comment:
According to the BEM methodology, blocks should not have external geometry styles like margin. It's better to move this style to a higher level or to a parent block.